### DISCUSSION TITLE
is Strongswan 5.7.2 compatible with Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by antonymsouza** [strongswan/strongswan#2466] [General]
We are planning to build docker image for Strongswan version 5.7.2 with Ubuntu 22.04 and other components like vpp+dpdk. is it possible to know about compatibility of Strongswan 5.7.2 can work with Ubuntu 22.04.

**Comment by tobiasbrunner** [comment]
Why? 5.7.2 was released in 2018, why would you use such an old version if you start a new project?

**Reply by antonymsouza** [reply]
Understand, thank you for the response. Will move to latest.
